core/service/fcm/container.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from dependency_injector import providers
+from dependency_injector.containers import DeclarativeContainer
+from firebase_admin import App, initialize_app
+from firebase_admin.credentials import Certificate
+
+from core.service.fcm.fcm import credentials
+
+if TYPE_CHECKING:
+    pass
```

Comment:
Do we need this for a side effect?

Reply:
Nope, just a leftover that should get cleaned up. I'll remove it.